src/databricks/labs/ucx/install.py

```diff
@@ -56,6 +56,36 @@
 ws = WorkspaceClient()
 
 print(__version__)
+
+# COMMAND ----------
+
+workspace_groups = [
+            g
+            for g in ws.groups.list(attributes='id,displayName,meta')
+            if g.meta.resource_type == "WorkspaceGroup"
+        ]
+print(f'groups: {{len(workspace_groups)}}')
+
+# COMMAND ----------
+
+from databricks.sdk.service import iam
+account_groups = [
+    iam.Group.from_dict(r)
+    for r in ws.api_client.do(
+        "get",
+        "/api/2.0/account/scim/v2/Groups",
+        query={{"attributes": "id,displayName,meta,members"}},
+    ).get("Resources", [])
+]
+account_groups = [g for g in account_groups if g.display_name not in ["users", "admins", "account users"]]
+print(f"Found {{len(account_groups)}} account groups")
+
+# COMMAND ----------
+
+ws_group_names = {{_.display_name for _ in workspace_groups}}
+ac_group_names = {{_.display_name for _ in account_groups}}
+group_names = list(ws_group_names.intersection(ac_group_names))
+print(f"Found {{len(group_names)}} groups to migrate")
```

Comment:
why do we need this in all debug nodebooks?

Reply:
I found it useful to have the possibility to check how many groups are still to be migrated (e.g. in case things are not working or fail in the middle of migration).

Reply:
imagine that someone doesn't know what groups migration is or not doing that anymore - does it add value for them in this debug notebook? maybe it should be in one of the markdown files in `docs/` folder?..

Reply:
sure I can move it under docs. I probably miss the overall vision of the tool. The local group migration is what this toll is currently used for.

Reply:
i moved it to the docs/local-group-migration.md